src/main/java/nextstep/courses/domain/cover/ImageExtension.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.courses.domain.cover;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public enum ImageExtension {
+    GIF("gif"),
+    JPG("jpg"),
+    JPEG("jpeg"),
+    PNG("png"),
+    SVG("svg"),
+    ;
+
+    private final String text;
+    ImageExtension(String text) {
```

Comment:
전반적으로 개행 컨벤션이 안 맞는 경우가 더러 있습니다.
커밋을 작은 단위로 진행해보시고 커밋할 때 마다 확인해 보시면 좋을 것 같아요
IDE의 reformat code 기능을 활용하시는 것을 추천드립니다.
```suggestion
    private final String text;
    
    ImageExtension(String text) {
```

Reply:
네 수시로 reformat 하도록 하겠습니다. 
커밋 작은 단위로 할 수 있도록 노력하겠습니다.